Formula/ignition-dome.rb

```diff
@@ -16,6 +16,10 @@ class IgnitionDome < Formula
     sha256 cellar: :any, mojave:   "25f52cf2b4893b2e98c3a9da733c33c434aa6eb9565f34bead4fb4eb725d6854"
   end
 
+  disable! date: "2022-01-31", because: "is past end-of-life date"
```

Comment:
adding a `disable!` tag automatically deprecates immediately

Reply:
* So you're saying that the date isn't respected? I was following what we did for Blueprint: https://github.com/osrf/homebrew-simulation/pull/1242

Reply:
* And now I see that was reverted in https://github.com/osrf/homebrew-simulation/pull/1256 :upside_down_face:

Reply:
https://github.com/osrf/homebrew-simulation/pull/1737